modules/basics/src/main/java/com/opengamma/strata/basics/StandardReferenceData.java

```diff
@@ -19,11 +22,11 @@
    */
   static final ImmutableReferenceData STANDARD;
   static {
-    ImmutableMap.Builder<ReferenceDataId<?>, Object> builder = ImmutableMap.builder();
+    Map<ReferenceDataId<?>, Object> map = new HashMap<>();
```

Comment:
Not related to the changes, why not `Map<ReferenceDataId<?>, HolidayCalendar>` ?

Also, same question for the next static{} block below